ledger/README.md

```diff
@@ -57,6 +57,52 @@ The Sandbox automatically makes all metrics available via JMX under the JMX doma
 When building an Indexer or Ledger API Server the implementer/ledger integrator is responsible to set up 
 a `MetricRegistry` and a suitable metric reporting strategy that fits their needs.
 
+## Health Checks
+
+### Ledger API Server health checks
+
+The Ledger API Server exposes health checks over the [GRPC Health Checking Protocol][]. You can check the health of
+the overall server by making a GRPC request to `grpc.health.v1.Health.Check`.
+
+You can also perform a streaming health check by making a request to `grpc.health.v1.Health.Watch`. The server will
+immediately send the current health of the Ledger API Server, and then send a new message whenever the health changes.
+
+The ledger may optionally expose health checks for underlying services and connections; the names of the services are
+ledger-dependent. For example, the Sandbox exposes two service health checks:
+
+- the `"index"` service tests the health of the connection to the index database
+- the `"write"` service tests the health of the connection to the ledger database
+
+To use these, make a request with the `service` field set to the name of the service. An unknown service name will
+result in a GRPC `NOT_FOUND` error.
+
+[GRPC Health Checking Protocol]: https://github.com/grpc/grpc/blob/master/doc/health-checking.md
+
+### Indexer health checks
+
+The Indexer does not currently run a GRPC server, and so does not expose any health checks on its own.
```

Comment:
```suggestion
The Indexer does not currently run a gRPC server, and so does not expose any health checks on its own.
```
Mostly for consistency with other mentions of gRPC.

Reply:
Their own documentation isn't consistent, which is why I was confused. 🙁